### PR TITLE
Create low volume dashboards from transactions explorer feed

### PR DESCRIPTION
**Pull request by mattrco**
This loads all records from the transactions explorer feed and,
depending on the data present, creates modules and a dashboard.

https://www.agileplannerapp.com/boards/15088/cards/9254

This also gives us a base to build on for updating all existing dashboards
from the feed. The spreadsheet merge test is disabled until we
need merge functions.

Overall this is fairly low-quality code, but it should suffice as a first iteration
for creating dashboards.

### CODE REVIEW

File: stagecraft/tools/import_dashboards.py [hunk 1]
```diff
@@ -0,0 +1,326 @@
+import logging
+import os
+import sys
+
+import requests
+
+from django.db import IntegrityError
+
+from spreadsheets import SpreadsheetMunger
```
Comment (jabley) [inline stagecraft/tools/import_dashboards.py]: `from .spreadsheets import SpreadsheetMunger`?

Comment (mattrco) [inline stagecraft/tools/import_dashboards.py]: Thanks, was sure I'd changed that...

File: stagecraft/tools/import_dashboards.py [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+        data for data in summaries if data['service_id'] == record['slug']]
+    quarterly_data = [
+        datum for datum in service_data if datum['type'] == 'quarterly']
+    seasonal_data = [
+        datum for datum in service_data \
+                if datum['type'] == 'seasonally-adjusted']
+
+    # Order in which modules are appended is significant
+    # as it will affect how the dashboard displays.
+    # TODO: for existing and new modules, ensure notes are updated.
```
Comment (jabley) [inline stagecraft/tools/import_dashboards.py]: Not sure who this TODO is for?

**Comment by jabley** [comment]
Otherwise, :+1:

**Comment by mattrco** [comment]
Thanks, updated. Also changed the criteria for creating modules after discussing with @tombooth and @easternbloc.

**Pull request merged**